core/migrations/0038_threshold_rename_storagemin_to_cpumin.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0032_auto_20150925_1239'),
+    ]
```

Comment:
@prosif - you're going to need to edit this `dependencies` for your migration at well. 

So fall in line with the rest of the migration, I believe this should be: 
```python
    dependencies = [
        ('core', '0037_add_end_date_identitymembership'),
    ]
```